app/models/title.rb

```diff
@@ -0,0 +1,3 @@
+class Title < ActiveRecord::Base
+  attr_accessible :name
+end
```

Comment:
Should title names be unique?

Reply:
Yes, I believe it should.. thanks, good catch!